public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,104 @@
+<!--
+  Heads up! 👋
+
+  Plugins:
+    - @tailwindcss/forms
+-->
+
+<div class="flex justify-center p-6 font-sans antialiased">
```

Comment:
???

Reply:
The styles for the container were not present in the preview, that's why I added this padding. Btw, is there a way to preview the change on dev mode (the tailwind classes were not loaded)

Reply:
After some research 🔬, I needed to add a container property in the frontmatter.

Reply:
Good stuff! You'd only need `p-6` as the rest is added for you